### PR TITLE
Docsify

### PR DESCRIPTION
**Pull request by fabiand**
gitbook was a good companion.

But now we move to docsify which is more self contained, and does not require any
server side automation.

This change set:
- Removes some old files
- Adds travics checks for unreferenced files
- Moves to docsify

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

**Comment by fabiand** [comment]
I need to rebase this onto current master.

**Comment by fabiand** [comment]
Ready for review

File: .aspell.en.pws [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+VirtualMachinePresets
+virtualmachinereplicaset
+VirtualMachineReplicaSet
+VirtualMachineReplicaSets
+virtualmachines
+VirtualMachines
+VirtualMachine's
+VirtualMachineSpec
+VirtualMachineTemplate
+Virutal
```
Comment (rwsu) [inline .aspell.en.pws]: remove

File: .aspell.en.pws [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+readOnly
+refererence
+registrydisk
+registryDisk
+RegistryDisk
+RegistryDisks
+registryvolume
+requiredDuringSchedulingIgnoredDuringExecution
+rhel
+roling
```
Comment (rwsu) [inline .aspell.en.pws]: remove

File: .aspell.en.pws [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+iSCSI
+JBNZuDzTZSF
+jUg
+KMGrS
+kRIc
+kube
+kubeconfig
+KUBECONFIG
+kubectl
+Kubernetes
```
Comment (rwsu) [inline .aspell.en.pws]: remove

File: .aspell.en.pws [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+ee
+ef
+eGHwAWO
+emptyDir
+emptydisk
+emptyDisk
+emptydiskvolume
+ENRjg
+ESo
+explicity
```
Comment (rwsu) [inline .aspell.en.pws]: remove

File: .aspell.en.pws (hunk 3 above)
Comment (fabiand) [reply .aspell.en.pws]: This needs to stay I suppose. As it is a correct and recognized word.

File: .aspell.en.pws (hunk 1 above)
Comment (fabiand) [reply .aspell.en.pws]: Fixed

File: .aspell.en.pws (hunk 2 above)
Comment (fabiand) [reply .aspell.en.pws]: Fixed

File: .aspell.en.pws (hunk 4 above)
Comment (fabiand) [reply .aspell.en.pws]: Isn't this also a correct word?

**Comment by fabiand** [comment]
Cool! Good to hear. :) I addressed your comments.
Thanks.

**Pull request merged**